code/game/objects/items/storage/boxes.dm

```diff
@@ -761,3 +761,211 @@
 	else if(!isopened)
 		isopened = 1
 		icon_state = "mealpackopened"
+
+//food boxes for storage in bulk
+
+//meat
+/obj/item/storage/box/meat
+	name = "box of meat"
+
+/obj/item/storage/box/meat/fill_preset_inventory()
+	for(var/i = 0, i < 7, i++)
+		new /obj/item/reagent_container/food/snacks/meat/monkey(src)
+
+//fish
+/obj/item/storage/box/fish
+	name = "box of fish"
+
+/obj/item/storage/box/fish/fill_preset_inventory()
+	for(var/i = 0, i < 7, i++)
```

Comment:
For loop as above.

Reply:
replace it by var/i in 1 to 7 like suggest thanks